test/handler.go

```diff
@@ -27,7 +27,7 @@ import (
 // to testing code for validation of proper transport using the Mixer API.
 type AttributesHandler interface {
 	// Check will be called once per Mixer API Check() request.
-	Check(attribute.Bag, *attribute.MutableBag) rpc.Status
+	Check(attribute.Bag, *attribute.MutableBag) (CheckResponse, rpc.Status)
```

Comment:
it doesn't matter for this at all, but just to be clear, the real Mixer code doesn't generate `CheckResponse` structs from Check() dispatches. This does provide a convenient override however.